std/assembly/typedarray.ts

```diff
@@ -1260,3 +1304,18 @@ function REVERSE<TArray extends ArrayBufferView, T>(array: TArray): TArray {
   }
   return array;
 }
+
+// @ts-ignore: decorator
+@inline
+function WRAP<TArray extends ArrayBufferView, T>(buffer: ArrayBuffer): TArray {
```

Comment:
Looks like this should not copy, but really just wrap the argument as the view's `data`. Will require allocating the view object and then set the respective fields etc.

Reply:
...possibly even taking optional `byteOffset = 0` and `byteLength = buffer.length` arguments so one can wrap a region of interest only.

Reply:
Fixed. Please let me know if `instantiate<TArray>(0)` is not the right way to allocate an empty `TArray`.

Reply:
Sorry didn't see the previous comment. Will change that

Reply:
Added optional arguments

Reply:
Best way would be to start with an `__alloc(sizeof<ArrayBufferView>(), idof<TArray>())` and go from there, validating and assigning data, dataStart and dataLength. Makes sure that no buffer is allocated at all.

Reply:
Fixed